README.md

```diff
@@ -31,3 +31,34 @@ Contributors
 Contributing
 ------------
 Please read more about [Github Flow](https://guides.github.com/introduction/flow/).
+
+Getting started
+---------------
+
+In order to customize docker skeleton:
+
+1. open `.project_name` file and set project name
```

Comment:
We could get rid of `.project_name` file in favor of [name attribute](https://github.com/PGSSoft/php-docker-skeleton/blob/master/composer.json#L2) which is loading [here](https://github.com/PGSSoft/php-docker-skeleton/blob/master/docker.sh#L9)